src/com/projectkorra/projectkorra/firebending/combo/JetBlaze.java

```diff
@@ -103,7 +103,7 @@ public void progress() {
 			fs.setDensity(8);
 			fs.setSpread(1.0F);
 			fs.setUseNewParticles(true);
-			fs.setCollisionRadius(3);
+			fs.setCollisionRadius(2);
```

Comment:
Couldn't this just be configurable?

Reply:
probably, but it wasnt initially, and also I know they are doing a big config update didn't want to mess that up

Reply:
Didn’t think those changes were going to be implemented until the next update, not this one.